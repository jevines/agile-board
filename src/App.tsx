import React from "react";
import Dashboard from "./screens/dashboard";
import { createGlobalStyle } from "styled-components";

const GlobalStyle = createGlobalStyle`
  body {
    height:100%;
  }
  html {
    height: 100%;
  }
`;

function App() {
  return (
    <div>
      <GlobalStyle />
      <Dashboard />
    </div>
  );
}

export default App;
