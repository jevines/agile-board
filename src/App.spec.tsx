import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

describe("The App comonent", () => {
  it("should match the snapshot", () => {
    const { container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
