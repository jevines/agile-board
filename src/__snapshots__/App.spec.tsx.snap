// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The App comonent should match the snapshot 1`] = `
<div>
  <section
    class="sc-pFZIQ hLaaHj"
  >
    <button
      class="sc-kEjbxe jAugHf"
      data-testid="addCardButton"
    >
      Click to add card
    </button>
    <section
      class="sc-jrAGrp jbfmIi"
    >
      <section
        class="sc-eCssSg foUHDZ"
      >
        <h1
          class="sc-jSgupP dhcVWK"
        >
          To Do
        </h1>
        <section
          class="sc-bdfBwQ bCQgUO"
        >
          <h2
            class="sc-gsTCUz gzfVJk"
          >
            mock task title
          </h2>
          <p
            class="sc-dlfnbm bglaea"
          >
            A mock description
          </p>
          <p
            class="sc-hKgILt fLjEpC"
          >
            comment 1
          </p>
          <p
            class="sc-hKgILt fLjEpC"
          >
            comment 2
          </p>
        </section>
      </section>
      <section
        class="sc-eCssSg foUHDZ"
      >
        <h1
          class="sc-jSgupP dhcVWK"
        >
          In Progress
        </h1>
        <section
          class="sc-bdfBwQ bCQgUO"
        >
          <h2
            class="sc-gsTCUz gzfVJk"
          >
            mock task title
          </h2>
          <p
            class="sc-dlfnbm bglaea"
          >
            A mock description
          </p>
          <p
            class="sc-hKgILt fLjEpC"
          >
            comment 1
          </p>
          <p
            class="sc-hKgILt fLjEpC"
          >
            comment 2
          </p>
        </section>
      </section>
      <section
        class="sc-eCssSg foUHDZ"
      >
        <h1
          class="sc-jSgupP dhcVWK"
        >
          QA
        </h1>
        <section
          class="sc-bdfBwQ bCQgUO"
        >
          <h2
            class="sc-gsTCUz gzfVJk"
          >
            mock task title
          </h2>
          <p
            class="sc-dlfnbm bglaea"
          >
            A mock description
          </p>
          <p
            class="sc-hKgILt fLjEpC"
          >
            comment 1
          </p>
          <p
            class="sc-hKgILt fLjEpC"
          >
            comment 2
          </p>
        </section>
      </section>
      <section
        class="sc-eCssSg foUHDZ"
      >
        <h1
          class="sc-jSgupP dhcVWK"
        >
          Done
        </h1>
        <section
          class="sc-bdfBwQ bCQgUO"
        >
          <h2
            class="sc-gsTCUz gzfVJk"
          >
            mock task title
          </h2>
          <p
            class="sc-dlfnbm bglaea"
          >
            A mock description
          </p>
          <p
            class="sc-hKgILt fLjEpC"
          >
            comment 1
          </p>
          <p
            class="sc-hKgILt fLjEpC"
          >
            comment 2
          </p>
        </section>
      </section>
    </section>
  </section>
</div>
`;
